### DISCUSSION TITLE
LaTeX Error: File `sourcesanspro.sty' not found.

### DISCUSSION THREAD

**Discussion by nirban256** [Cyfrin/security-and-auditing-full-course-s23#59] [Q&A] [answered]
I am having trouble while using `pandoc` command to generate pdf from my `report.md`.
Whenever I am wrting the following command it is showing me the following error :-
```
pandoc report.md -o report.pdf --from markdown --template=eisvogel --listings
```
The error is:
```
Error producing PDF.
! LaTeX Error: File `sourcesanspro.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

Enter file name: 
! Emergency stop.
<read *> 
         
l.179   \usepackage
```
I am writing code in wsl on top of windows 11, what am i doing wrong here?
Can anyone help me !

**Comment by rube-de** [comment]
If you installed a minimal version of LaTeX, like TinyTeX or BasicTeX, and have missing .sty sources you can install them via tlmgr
e.g.
```
tlmgr install sourcecodepro
```

this is also written on the pandoc installation page.

**Reply by cromewar** [reply]
Amazing @rube-de, does this solution works for you @nirban256?

**Reply by nirban256** [reply]
I am facing this issue after running the above command 
```
(running on Debian, switching to user mode!)
(see /usr/share/doc/texlive-base/README.tlmgr-on-Debian.md)
TLPDB: not a directory, not loading: /home/nirban/texmf
tlmgr: user mode not initialized, please read the documentation!
```
What do I need to do now?

**Reply by rube-de** [reply]
oh yeah I'm on mac. According to quick search it seems you need to initialize user mode on linux for the tex tlive manager (tlmgr),
but the .md actually discourage from this, see https://askubuntu.com/a/1489306

**Reply by nirban256** [reply]
ok

**Reply by nirban256** [reply]
I have read the documentation but I couldn't understand clearly what to do as I am new to linux. What do I need to do here?

**Reply by kwamentw** [reply, marked as answer]
It's normal to have that, you have to initialize it with this command
 `tlmgr init-usertree`

**Reply by nirban256** [reply]
thank you very much for your help

**Comment by kwamentw** [comment]
@nirban256 I hope this works

**Comment by 0xbadal** [comment]
download manually all missing pakeges from MikTeX console
